### ISSUE TITLE
content not available on this app

### ISSUE THREAD

**Issue by DSDV** [anxdpanic/plugin.video.youtube#493]
### Context

Please provide any relevant information about your setup

* Add-on Version: latest from repo
* Kodi Version: 21
* Kodi GUI Language: english
* Operating System: coreelec nightly
* Operating System Language: english

------

### Expected Behavior

working youtube trailers

------

### Current Behavior

What is the current behavior?

------

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1.  install addon
2.  set api keys in webinterface
3.  try playback

------

### Additional Information

it is a fresh coreelec setup and i just followed the popup promt which than lead to a dead youtube tutorial. 
gooled the linked tutorial how to get my keys (fyi linked tutorial is out of date the website looks really diffrent). 
entered keys from the webgui as mentioned in the wiki. 

got the error :( 

in contrast to other issues mentioning this error it never worked for me in the first place :/ 

i would appreciate any help :) 

------

**Comment by mYnDstrEAm** [comment]
Same problem here. The addon stopped working and trailers can't be watched anymore. I don't entered any api keys and users should not be required to set up api keys. It used to work before.

I get this error with Kodi 19 on Debian 11 (`Running: YouTube (6.8.25+matrix.1) on Matrix (Kodi-19.1) with Python 3.9.2`): 

```
Path: /
Params: {'action': 'play_video', 'videoid': '{id}'}
WARNING <general>: [plugin.video.youtube] DEPRECATED "plugin://plugin.video.youtube/?action=play_video&videoid={id}"
WARNING <general>: [plugin.video.youtube] USE INSTEAD "plugin://plugin.video.youtube/play/?video_id={id}"
ERROR <general>: Traceback (most recent call last):
ERROR <general>: 
ERROR <general>:   File "/home/username/.kodi/addons/plugin.video.youtube/resources/lib/youtube_plugin/youtube/helper/yt_play.py", line 50, in play_video
                                                       video_streams = client.get_video_streams(context, video_id)
...
ERROR <general>: youtube_plugin.youtube.youtube_exceptions.YouTubeException: The following content is not available on this app.
```
I replaced the video Id with {id} above.

And I get the same error with Kodi 20 on Debian 12 (`Running: YouTube (7.0.1) on N***** (Kodi-20.1) with Python 3.11.2`) (and why is it called `N*****` there, is it supposed to be like that?).

**Comment by RalfJung** [comment]
Duplicate of https://github.com/anxdpanic/plugin.video.youtube/issues/442

**Comment by AchillesPunks** [comment]
Has anyone found a Fix for this error yet?  

I just got it working

"The following content is not available on this app"

**Comment by MoojMidge** [comment]
Duplicate of #442

**Comment by MoojMidge** [comment]
Fix is in the first post of #442 and available in the development release:
https://github.com/anxdpanic/plugin.video.youtube/releases/tag/7.x.x-dev
